testing/unittests/cryptotests.cpp

```diff
@@ -299,50 +299,45 @@ class CryptoUnitTest : public CppUnit::TestFixture
             ASSERT(ok);
 
             //Perform
-            printf("digiSign() loop test\n");
+            DBGLOG("digiSign() loop test");
             unsigned now = msTick();
             for (int x=0; x<1000; x++)
             {
                 dsm->digiSign(sig3.clear(), text3);
             }
-            printf("digiSign() 1000 iterations took %d MS\n", msTick() - now);
+            DBGLOG("digiSign() 1000 iterations took %d ms", msTick() - now);
 
-            printf("digiVerify() loop test\n");
+            DBGLOG("digiVerify() loop test");
             now = msTick();
             for (int x=0; x<1000; x++)
             {
                 dsm->digiVerify(sig3, text3);
             }
-            printf("digiverify 1000 iterations took %d MS\n", msTick() - now);
+            DBGLOG("digiverify 1000 iterations took %d ms", msTick() - now);
 
             now = msTick();
-            printf("\nAsynchronous test digiSign\n");
+            DBGLOG("Asynchronous test digiSign");
             asyncDigiSignUnitTest(dsm);
-            printf("digiSign 1000 async iterations took %d MS\n", msTick() - now);
+            DBGLOG("digiSign 1000 async iterations took %d ms", msTick() - now);
 
             now = msTick();
-            printf("\nAsynchronous test digiVerify\n");
+            DBGLOG("Asynchronous test digiVerify");
             asyncDigiVerifyUnitTest(dsm);
-            printf("digiverify 1000 async iterations took %d MS\n", msTick() - now);
+            DBGLOG("digiverify 1000 async iterations took %d ms", msTick() - now);
 
             now = msTick();
-            printf("\nAsynchronous test digiSign and digiVerify\n");
+            DBGLOG("Asynchronous test digiSign and digiVerify");
             asyncDigiSignAndVerifyUnitTest(dsm);
-            printf("digiSign/digiverify 1000 async iterations took %d MS\n", msTick() - now);
+            DBGLOG("digiSign/digiverify 1000 async iterations took %d ms", msTick() - now);
         }
         catch (IException *e)
         {
             StringBuffer err;
             e->errorMessage(err);
-            printf("Digisign IException thrown:%s\n", err.str());
-            exception.setown(e);
-        }
-        catch (CppUnit::Exception &e)
-        {
-            printf("Digisign CppUnit::Exception thrown\n");
-            cppunitException = e.clone();
```

Comment:
Should also delete the variable definition on line 228.

Reply:
Fixed and repushed